src/SwedbankPay.Sdk.Infrastructure/PayeeInfoResponse.cs

```diff
@@ -0,0 +1,46 @@
+﻿using SwedbankPay.Sdk.PaymentOrders;
+using System;
+
+namespace SwedbankPay.Sdk
+{
+    internal class PayeeInfoResponse : Identifiable, IPayeeInfo
+    {
+        public PayeeInfoResponse(Uri id) : base(id)
+        {
+        }
+
+        public PayeeInfoResponse(PayeeInfoResponseDto dto)
+            :base(dto.Id)
+        {
+            OrderReference = dto.OrderReference;
+            PayeeId = dto.PayeeId;
+            PayeeName = dto.PayeeName;
+            PayeeReference = dto.PayeeReference;
+            ProductCategory = dto.ProductCategory;
+            Subsite = dto.Subsite;
+        }
+
+        public PayeeInfoResponse(PaymentOrderPayeeInfoDto dto)
+            : base(dto.Id)
+        {
+            if (string.IsNullOrEmpty(dto.PayeeId))
+            {
+                return;
+            }

```

Comment:
Is this `if` necessary? If all properties of the DTO is `null`, the properties set below will have the same result as not setting anything at all.

```suggestion
```

Reply:
This is true. It's a leftover from when not everything was a `string`, to avoid null reference exceptions.